lib/compiler/assets-manager.ts

```diff
@@ -13,27 +13,9 @@ import { getValueOrDefault } from './helpers/get-value-or-default';
 export class AssetsManager {
   private watchAssetsKeyValue: { [key: string]: boolean } = {};
   private watchers: chokidar.FSWatcher[] = [];
-  private actionInProgress = false;
 
-  /**
-   * Using on `nest build` to close file watch or the build process will not end
-   * Interval like process
-   * If no action has been taken recently close watchers
-   * If action has been taken recently flag and try again
-   */
   public closeWatchers() {
-    // Consider adjusting this for larger files
-    const timeoutMs = 500;
-    const closeFn = () => {
-      if (this.actionInProgress) {
-        this.actionInProgress = false;
-        setTimeout(closeFn, timeoutMs);
-      } else {
-        this.watchers.forEach((watcher) => watcher.close());
-      }
-    };
-
-    setTimeout(closeFn, timeoutMs);
+    this.watchers.forEach((watcher) => watcher.close());
```

Comment:
```suggestion
    this.watchers.forEach((watcher) => watcher.close());
```
AFAIR `close()` returns a Promise

Reply:
It's the same as before. Is there a problem with that?